Tests/UnitTests/Screens/Home/HomeViewControllerTests.swift

```diff
@@ -22,16 +22,25 @@ final class HomeTabViewModelTests: XCTestCase {
     }
 }
 
-extension HomeTabViewModelTests {
+extension HomeViewControllerTests {
     func test_title_contents() throws {
```

Comment:
please can we add tests for the `UITableViewDataSource` methods?

Reply:
Yep, there's a test for each method we override now, let me know if you think more could be done 👍